mne_bids/write.py

```diff
@@ -795,7 +836,8 @@ def make_dataset_description(path, name, data_license=None,
     """
     # default author to make dataset description BIDS compliant
     if authors is None:
-        authors = "MNE-BIDS"
+        authors = ("For publications please add the MNE-BIDS citations in "
+                   "the README to your manuscript before removing this")
```

Comment:
in the README to your manuscript? I would suggest removing "in the readme". People should spend some effort, we can't spoon feed everything :-)

Reply:
I feel like if you spoon feed anyone something it should be the citation to your paper...

Reply:
I changed the wording though so it's less ambiguous, thanks for catching that